### ISSUE TITLE
getPedMoveState returns incorrect move state while entering a vehicle

### ISSUE THREAD

**Issue by xLive** [multitheftauto/mtasa-blue#1594]
**Describe the bug**
[getPedMoveState](https://wiki.multitheftauto.com/wiki/GetPedMoveState) returns incorrect state while entering a vehicle
like powerwalk, fall

**To reproduce**
1. enter a vehicle 
2. while entering a vehicle 
do crun getPedMoveState(localPlayer)
3. you will probably see "powerwalk" or "fall"

or you can use a script I made it to detect any [getPedMoveState](https://wiki.multitheftauto.com/wiki/GetPedMoveState) changes and output it to the chatbox
[moveState.zip](https://github.com/multitheftauto/mtasa-blue/files/5018167/moveState.zip)

**Expected behaviour**
it should return false or specific state

**Screenshots**

![mta-screen_2020-08-03_22-34-40](https://user-images.githubusercontent.com/30420446/89222117-21379180-d5dd-11ea-8556-d4bfea00e503.png)
**Version**
v1.5.7-release-20582

**Comment by Unde-R** [comment]
when leaving the vehicle It returns false too.

**Comment by xLive** [comment]
> when leaving the vehicle It returns false too.

Because you still in the vehicle
this is what the function returns if you tried to use it in a vehicle

**Comment by Unde-R** [comment]
> > when leaving the vehicle It returns false too.
> 
> Because you still in the vehicle
> this is what the function returns if you tried to use it in a vehicle

Yeah now it returns leavingvehicle.

**Comment by Unde-R** [comment]
and enteringvehicle.

**Comment by qaisjp** [comment]
What does it return when the player is entering a vehicle, but the vehicle drives off a little, so the player ends up running towards the vehicle? Or when the player is walking to the passenger door?

I think it would be good to return the correct walk/powerwalk/sprint etc in these scenarios.

**Naming**

We currently have "stand", "walk", "jog", "sprint", etc. So "enteringvehicle" is not using the correct tense.

**Vehicle-related movement states**

Note that `getPedMoveState` documents what happens when the player is in a vehicle:

> Returns a string indicating the ped's move state, or **false if** the ped is not streamed in, the movement type is unknown, **the ped is in a vehicle** or the ped is invalid.

Basically, I'm not 100% sure whether `enter_vehicle` and `leave_vehicle` fits well with the rest of these move states. What do you think?

What does this function return when we're flying in a jetpack or parachuting? (I'm guessing it returns `fall`?)

**Comment by Unde-R** [comment]
> What does it return when the player is entering a vehicle, but the vehicle drives off a little, so the player ends up running towards the vehicle? Or when the player is walking to the passenger door?
> 
> I think it would be good to return the correct walk/powerwalk/sprint etc in these scenarios.
> 
> **Naming**
> 
> We currently have "stand", "walk", "jog", "sprint", etc. So "enteringvehicle" is not using the correct tense.
> 
> **Vehicle-related movement states**
> 
> Note that `getPedMoveState` documents what happens when the player is in a vehicle:
> 
> > Returns a string indicating the ped's move state, or **false if** the ped is not streamed in, the movement type is unknown, **the ped is in a vehicle** or the ped is invalid.
> 
> Basically, I'm not 100% sure whether `enter_vehicle` and `leave_vehicle` fits well with the rest of these move states. What do you think?
> 
> What does this function return when we're flying in a jetpack or parachuting? (I'm guessing it returns `fall`?)

let me try

**Comment by Unde-R** [comment]
while flying in a jetpack or parachuting it returns fall, while trying to enter a car or walking to the passenger door it returns enteringvehicle.

**Comment by Unde-R** [comment]
> Since we currently have "stand", "walk", "jog", "sprint", etc. So "enteringvehicle" is not using the correct tense.

I'll just make it return false, instead of "fall" when entering or leaving
